db/migrate/20150120095012_create_categories.rb

```diff
@@ -2,7 +2,7 @@ class CreateCategories < ActiveRecord::Migration
   def change
     create_table :categories do |t|
       t.string :name
-      t.string :link
+      t.string :simplified_name
       
```

Comment:
Better to do it now otherwise you'll have to drop and migrate the db again, or to add it in a separate migration file later